### PR TITLE
CUMULUS-2578: Add PUT /executions endpoint to update an execution

### PR DESCRIPTION
**Pull request by jennyhliu**

### CODE REVIEW

File: content/executions.md [hunk 1]
```diff
@@ -425,6 +425,69 @@ POST /executions
 
 ```curl
 $ curl --request POST https://example.com/executions --header 'Authorization: Bearer ReplaceWithToken' --header 'Content-Type: application/json' --data '{
+    "arn": "arn:aws:states:us-east-1:123456789012:execution:TestSrcIntegrationIngestGranuleStateMachine-UhCSmszl2sgv:cff1266e-ef36-664f-a649-3a4d26bd1735",
```
Comment (flamingbear) [inline content/executions.md:428]: It looks like the schema above is incorrect now?
Missing createAt and updatedAt.  Do you mind updating those?

**Review by flamingbear — APPROVED** [review]
One thing that wasn't your fault, but I'm asking you to fix anyway. :) 
But I'll approve now so you don't have to wait on me.

File: content/executions.md (hunk 1 above)
Comment (jennyhliu) [reply content/executions.md:428]: createdAt and updatedAt are added when the execution is inserted into database.

Comment (flamingbear) [reply content/executions.md:428]: Oh, then I probably need to change my create code.

**Pull request merged**